src/darker/tests/test_command_line.py

```diff
@@ -279,23 +265,21 @@ def test_parse_command_line(
             "check": True,
             "isort": True,
             "lint": ["pylint"],
-            "skip_string_normalization": True,
-            "skip_magic_trailing_comma": True,
             "line_length": 88,
             "target_version": "py37",
         },
-        expect_warn={
-            "The configuration option `skip_magic_trailing_comma` in [tool.darker] is"
-            " deprecated and will be removed in Darker 3.0.",
-            "The configuration option `skip_string_normalization` in [tool.darker] is"
-            " deprecated and will be removed in Darker 3.0.",
```

Comment:
I would love for the information that this is moved to `[tool.black]` now to be included here, so I don't have to research that when getting the warning.